weight-meter/src/lib.rs

```diff
@@ -2,6 +2,9 @@
 
 use frame_support::weights::Weight;
 
+// For use in mock file
+extern crate self as orml_weight_meter;
```

Comment:
interesting you can do this why is it needed here? how about `use crate as orml_weight_meter` in mock.rs?

Reply:
That's what I had originally tried but for some reason I couldn't get it to work.